repoindex.md

```diff
@@ -261,6 +261,34 @@ organization in GitHub under
 		</tr>
 	</table>
 </fieldset>
+<fieldset class="repo">
+	<legend>
+		pmem-related workloads
+	</legend>
+	<p>
+	These repos contain non-volatile memory related workloads.
+	Their concept is based on combined market ready solutions with
+	persistent memory. Workloads are delivered in SaaC (Solution as
+	a Code) concept, in the form of bash/ansible scripts operating
+	on the provided infrastructure. The infrastructure requirements are
+	described in the documentation (README.md). They use multiplication
```

Comment:
I don't understand what this last sentence means. What is multiplication in this context?

Reply:
also dropped